.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [ 16, 18, 19, 20 ]
+        node: [ 16, 18, 19, 20, 21, 22 ]
```

Comment:
Have we tested with the other node versions?

Reply:
The CI _should_ be sufficient for this case, as the unit tests pass and the test page code is all served statically, with exception to the kvs-webrtc(.min).js which is the Signaling SDK code that gets bundled through Webpack.